### ISSUE TITLE
'NaN' number in Output Action of the neuron network

### ISSUE THREAD

**Issue by odannis** [Unity-Technologies/ml-agents#2115]
Hi,

After the training of an agent with a visual observation with an LSTM, i run inference with the brain obtain. After several steps, the agent freeze and the code seems to be in a loop.

I managed to extract all of the layers from the network. I observed that  'NaN' appears in the layer "main_graph_0_encoder0_conv_1_BiasAdd" which propagates into the layer action (Tensor action = [NaN, NaN, NaN, NaN, NaN]).  I think it is the NaN which cause the freezing of the agent ? 

I join the Matlab file containing all the layers of the network and the parameter of my brain.

[ReseauDeNeurones.zip](https://github.com/Unity-Technologies/ml-agents/files/3272561/ReseauDeNeurones.zip)

Thank you for your help.

trainer | ppo
-- | --
batch_size | 2048
beta | 0.01
buffer_size | 4096
epsilon | 0.2
gamma | 0.995
hidden_units | 256
lambd | 0.98
learning_rate | 0.0003
max_steps | 5.0e6
memory_size | 256
normalize | False
num_epoch | 5
num_layers | 3
time_horizon | 128
sequence_length | 64
summary_freq | 1000
use_recurrent | True
use_curiosity | False
curiosity_strength | 0.01
curiosity_enc_size | 256
summary_path | ./summariesANDO/MazeVisualResetMaterials5-0_MazeTVisualBrain
model_path | ./models/MazeVisualResetMaterials5-0/MazeTVisualBrain
keep_checkpoints | 5
